app/Repositories/BlogRepository.php

```diff
@@ -32,17 +31,18 @@ public function deleteById(int $id): bool
 
     public function getRelatedBlog(int $id): Collection
     {
-        return Blog::with('user')->whereHas('user', function ($q) {
-            $q->where('status', User::STATUS_VERIFIED);
-        })
-        ->inRandomOrder()->approved()->where('id', '!=', $id)->limit(config('constant.limit'))->get();
+        return Blog::with('user')
+            ->whereRelation('user', 'status', '=', User::STATUS_VERIFIED)
```

Comment:
https://stackoverflow.com/questions/41679771/eager-load-relationships-in-laravel-with-conditions-on-the-relation
đừng dùng whereRelation